src/main/java/com/atlauncher/data/Settings.java

```diff
@@ -630,6 +631,8 @@ public void loadStartingProperties() {
             this.enableTrayIcon = Boolean.parseBoolean(properties.getProperty("enabletrayicon", "true"));
             this.enableDiscordIntegration = Boolean
                     .parseBoolean(properties.getProperty("enablediscordintegration", "true"));
+            this.enableFeralGamemode = Boolean
+                    .parseBoolean(properties.getProperty("enableferalgamemode", "true"));
```

Comment:
Would be better to default these to `Utils.executableInPath("gamemoderun")` right?

Reply:
Yes good idea.

Reply:
Done.